crates/cargo-bsp/src/project_model/target_details.rs

```diff
@@ -1,29 +1,33 @@
+use cargo_metadata::camino::Utf8PathBuf;
 use std::collections::BTreeSet;
 
-use cargo_metadata::camino::Utf8PathBuf;
+use crate::project_model::build_target_mappings::parent_path;
 use log::error;
 use serde_enum_str::{Deserialize_enum_str, Serialize_enum_str};
 
-use crate::project_model::build_target_mappings::parent_path;
 use crate::project_model::cargo_package::{CargoPackage, Feature};
 
-#[derive(Debug, Clone)]
-pub struct TargetDetails<'a> {
+#[derive(Debug, Clone, Default)]
+pub struct TargetDetails {
     pub name: String,
     pub kind: CargoTargetKind,
     pub package_abs_path: Utf8PathBuf,
+    pub package_name: String,
     pub default_features_disabled: bool,
-    pub enabled_features: &'a BTreeSet<Feature>,
+    pub enabled_features: BTreeSet<Feature>,
 }
 
-impl<'a> TargetDetails<'a> {
+const FEATURE_FLAG: &str = "--feature ";
```

Comment:
to chyba nie miejsce na to i funkcję, która z tego korzysta. Przeniósłbym tę stałą do komendy i już tam bym dodawał albo całą funkcję, do miejsca,w którym się tworzy komendę